package/README.md

```diff
@@ -71,6 +71,17 @@ You can then use that `getClient` function in your server components:
 const { data } = await getClient().query({ query: userQuery });
 ```
 
+#### Multiple clients
+
+If you need to manage multiple clients, for example if you have different endpoints for content previews,
+the function you pass to `registerApolloClient` can accept arbitrary arguments.
+If you're using Typescript, you can hint their types as a tuple:
+
+```ts
+export const { getClient } = registerApolloClient<[string | null]>((previewToken: string | null) => {
```

Comment:
```suggestion
export const { getClient } = registerApolloClient((token: string | null) => {
```

Couple things here:

1. Let's rely on type inference where possible since its easier to understand than passing the tuple as a generic. We find a lot of people will copy/paste code, so we tend to prefer showing off the "optimal" code when possible. I've pulled this down locally and this change works as expected.
2. The name `previewToken` feels a bit specific for the README. Could we make this a tad more generic to something like `token` or `authToken` which are more wide-spread?